ipinfo/cmd_tool_is_global_unicast.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/ipinfo/cli/lib"
+	"github.com/ipinfo/cli/lib/complete"
+	"github.com/ipinfo/cli/lib/complete/predict"
+	"github.com/spf13/pflag"
+)
+
+var completionsToolIsGlobalUnicast = &complete.Command{
+	Flags: map[string]complete.Predictor{
+		"-h":      predict.Nothing,
+		"--help":  predict.Nothing,
+		"-q":      predict.Nothing,
+		"--quiet": predict.Nothing,
+	},
+}
+
+func printHelpToolIsGlobalUnicast() {
+	fmt.Printf(
+		`Usage: %s tool is_global_unicast [<opts>] <cidr | ip | ip-range | filepath>
+
+Description:
+  Checks if the input is a global unicast address.
+  Inputs can be IPs, IP ranges, CIDRs, or filepath to a file
+
+Examples:
+  $ %[1]s tool is_global_unicast 10.255.0.0 | 2000::1
+  $ %[1]s tool is_global_unicast 255.255.255.255 | ff00::1
```

Comment:
Still have false examples

Reply:
sorry about that just forgot to remove the examples the rest should be okay now please check @UmanShahzad